integration/pod_userns_linux_test.go

```diff
@@ -304,6 +304,98 @@ func TestPodUserNS(t *testing.T) {
 	}
 }
 
+func TestIssue10598(t *testing.T) {
```

Comment:
I guess you have verified this test fails before the patch, right? As there are several subtle things, I'd like to confirm this.